iset.mm

```diff
@@ -116375,6 +116354,33 @@ Metamath Proof Explorer (aka "set.mm").  This list of conventions is
        a decidability condition added should add "dc" to the theorem name.
        A theorem in set.mm where "nonempty class" is changed to "inhabited
        class" should add "m" (for member) to the theorem name.</li>
+
+       <li><b>iset.mm versus set.mm names</b>
+       <p>Theorems which are the same as in set.mm should be named the same
+       (that is, where the statement of the theorem is the same; the proof
+       can differ without a new name being called for).  Theorems which are
+       different should be named differently (we do have a small number of
+       intentional exceptions to this rule but on the whole it serves us
+       well).</p>
+
+       <p>As for how to choose names so they are different between iset.mm
+       and set.mm, when possible choose a name which reflect the difference
+       in the theorems. For example, if a theorem in set.mm is an equality
+       and the iset.mm analogue is a subset, add "ss" to the iset.mm name.
+       If need be, add "i" to the iset.mm name (usually as a prefix to
+       some portion of the name).</p>
+
+       <p>As with set.mm, we welcome suggestions for better names (such
+       as names which are more consistent with naming conventions).</p>
+
+       <p>We do try to keep set.mm and iset.mm similar where we can. For
+       example, if a theorem exists both places but the name in set.mm
```

Comment:
typo
...exists in both places...

Reply:
I think either way is grammatical but :shrug: I guess "in both places" is a little better. Fixed.